projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/notices.js

```diff
@@ -175,14 +187,20 @@ function GlobalStylesEditNotice() {
 			} );
 		}
 
+		if ( isSiteEditor ) {
+			actions.push( {
+				label: __( 'Remove premium styles', 'jetpack-mu-wpcom' ),
+				onClick: resetGlobalStyles,
+				variant: 'secondary',
+				noDefaultClasses: true,
+			} );
+		}
+
 		actions.push( {
-			label: __( 'Remove premium styles', 'jetpack-mu-wpcom' ),
-			onClick: isSiteEditor ? resetGlobalStyles : openResetGlobalStylesSupport,
-			variant: isSiteEditor ? 'secondary' : 'link',
+			label: __( 'Learn more', 'jetpack-mu-wpcom' ),
+			onClick: openLearnMoreAboutStylesDialog,
+			variant: 'link',
 			noDefaultClasses: true,
-			className: isSiteEditor
-				? ''
-				: 'wpcom-global-styles-action-has-icon wpcom-global-styles-action-is-external wpcom-global-styles-action-is-support',
```

Comment:
The `wpcom-global-styles-action-is-support` class is no longer used, so we can clean up some unused CSS code now:

https://github.com/Automattic/jetpack/blob/671d537286be54d41935a1db6991e8645aa74a77/projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/notice.scss#L80-L87

Reply:
Thanks for pointing that out, I have removed that unused style in this commit [4e640eb](https://github.com/Automattic/jetpack/pull/39226/commits/4e640eb2777c05860a752de6bcd97ab20c82fd2d)